var/spack/repos/builtin/packages/cxx/package.py

```diff
@@ -34,8 +35,13 @@ def test(self):
             cxx_opts = [compiler.cxx11_flag] if "c++11" in test else []
 
             cxx_opts += ["-o", exe_name, filepath]
-            compiled = self.run_test(cxx_exe, options=cxx_opts, installed=True)
+            cxx_exe = which(join_path(self.prefix.bin, cxx_exe))
+            compiled = cxx_exe(*cxx_opts)
```

Comment:
This is not the conversion for `run_test()`. 

Also, keep in mind that this is an iteration on multiple custom test files (https://github.com/spack/spack/tree/develop/var/spack/repos/builtin/packages/cxx/test) and each should be treated separately (https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-stand-alone-test-parts).

Reply:
Aside from adding `test_part` is there something else thats wrong?